### DISCUSSION TITLE
How should I fill in the values of these variables (bondville, single point)

### DISCUSSION THREAD

**Discussion by Zhoujieliang** [NCAR/hrldas#153] [Q&A] [answered]
![image](https://github.com/NCAR/hrldas/assets/125653764/99a0f443-07f0-4680-a1d4-3f3febaa7d34)
I seem to be able to only find the longitude and latitude, vegetation type, soil type, and altitude of Bondville. Where can I obtain additional variable information?

**Comment by Zhoujieliang** [comment]
![bondville](https://github.com/NCAR/hrldas/assets/125653764/7a91aa20-8691-4a4a-a9cf-61e9de96f1b7)

**Comment by Zhoujieliang** [comment]
such as maximum_vegetation_pct or canopy_water, snow_water_equivalent

**Comment by cenlinhe** [comment]
@CharlesZheZhang Could you please take a look at this?

**Comment by CharlesZheZhang** [comment]
Hi Cenlin,

I will reply to this discussion later today.

Best,
Zhe


On Fri, Jan 12, 2024 at 9:39 AM Cenlin_He ***@***.***> wrote:

> @CharlesZheZhang <https://github.com/CharlesZheZhang> Could you please
> take a look at this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/hrldas/discussions/153#discussioncomment-8112167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHANQAD745QYYRRF3JM3JVLYOFRL7AVCNFSM6AAAAABBXTHJYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMJSGE3DO>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Zhe

**Comment by CharlesZheZhang** [comment, marked as answer]
Hi @Zhoujieliang , this bondville.dat provides a template for testing Noah-MP at single site. Originally, these values are filled from observation data. If you have these values provided from any observation, then you can fill them in. 

Another way to obtain them is that, if you are running this single site for many years for spin-up, so you can run this one year data for multiple years, (change the loop option in namelist). So you can start with an initial value and the model should reach an equilibrium state after multiple years. 

Hope this can help.
Zhe

**Reply by Zhoujieliang** [reply]
Thanks

**Comment by tslin2** [comment]
For maximum_vegetation_pct and minimum_vegetation_pct, 

There is monthly data available from the WPS geogrid preprocessing tool (variable name: GREENFRAC in WPS output), so you can also get from there.

@CharlesZheZhang @cenlinhe Please correct me if I am wrong.

**Reply by cenlinhe** [reply]
Yes, it is correct. SHDMAX and SHDMIN give the max and min of vegetation fraction/percentage for each grid.

**Comment by tslin2** [comment]
@CharlesZheZhang @cenlinhe 
If I am not wrong, there is nowhere to use minimum_vegetation_pct (SHDMIN) in the Noah-MP model. 
Why do we need this input, or can it be removed in the future? Or am I missing something?

**Reply by cenlinhe** [reply]
you are right that SHDMIN is not currently used in Noah-MP. It is just providing users with a way to read in SHDMIN to allow them to test with the use of SHDMIN if they want. I think we can still keep it there for potential future use.
